packages/smooth_app/lib/database/local_database.dart

```diff
@@ -73,6 +65,18 @@ class LocalDatabase extends ChangeNotifier {
     return localDatabase;
   }
 
+  static Future<String> _getDatabasePath() async {
+    final String databasesRootPath;
+    if (defaultTargetPlatform == TargetPlatform.iOS) {
+      // as suggested in https://pub.dev/documentation/sqflite/latest/sqflite/getDatabasesPath.html
+      final Directory directory = await getLibraryDirectory();
+      databasesRootPath = directory.path;
+    } else {
+      databasesRootPath = await getDatabasesPath();
+    }
+    return join(databasesRootPath, 'smoothie.db');
```

Comment:
The db name should be a constant in your class.
The same goes for the DB version

Reply:
I guess this is something that was already there not my written code, i can change that to so,but was not of a big road breaker